packages/ui-react/src/components/Shelf/Shelf.module.scss

```diff
@@ -32,12 +32,11 @@
   padding: 12px 0;
   outline-color: var(--highlight-color, white);
   cursor: pointer;
-  opacity: 0;
```

Comment:
Always show the chevron buttons. This has impact for all users. I could not find a solution to only show the buttons when the screen reader pointer has this button focussed. This would be preferred.

Reply:
I think there is no other way indeed.